src/Client.php

```diff
@@ -337,6 +337,26 @@ public function waitFor(string $locator, int $timeoutInSecond = 30, int $interva
         return $this->crawler = $this->createCrawler();
     }
 
+    /**
+     * @param string $locator
```

Comment:
Please remove all the `params` (we only add them when they provide additional information, and it's not the case here).

Reply:
This param is identical in purpose to the first argument of `waitFor()`. In the case of `waitFor()`, a parameter description is added to the docblock to make it clear that this can be either a CSS selector or XPath expression.

Should the same be added here instead of removing this param?

Reply:
@dunglas @webignition i update the phpdoc of `waitForVisibility()` function to be the same of `waitfFor()` .